src/nvim/eval.c

```diff
@@ -15219,10 +15219,16 @@ static void f_setline(typval_T *argvars, typval_T *rettv)
 /*
  * Used by "setqflist()" and "setloclist()" functions
  */
-static void set_qf_ll_list(win_T *wp, typval_T *list_arg, typval_T *action_arg, typval_T *rettv)
+static void set_qf_ll_list(
```

Comment:
Style guide require moving all arguments to the new line *only* if you have no other choice. `win_T *wp` should be here and not on next line. Also missing function documentation and function attributes.